auth-tokens/src/main/java/com/palantir/tokens/auth/BearerToken.java

```diff
@@ -71,10 +69,13 @@
 
     @JsonCreator
     public static BearerToken valueOf(String token) {
-        Preconditions.checkArgument(token != null, "BearerToken cannot be null");
-        Preconditions.checkArgument(!token.isEmpty(), "BearerToken cannot be empty");
-        if (!isValidBearerToken(token)) {
-            throw new SafeIllegalArgumentException("BearerToken must match pattern",
+        if (token == null) {
+            throw new MalformedTokenException("Bearer token cannot be null");
+        } else if (token.isEmpty()) {
+            throw new MalformedTokenException("Bearer token cannot be empty");
+        } else if (!isValidBearerToken(token)) {
+            throw new MalformedTokenException(
+                    "Bearer token must match pattern",
```

Comment:
May want to avoid modifying these messages in this change to minimize potential failures from message equality checks (we've had a few of those lately).